packages/pyright-internal/src/common/chokidarFileWatcherProvider.ts

```diff
@@ -44,7 +44,7 @@ export class ChokidarFileWatcherProvider implements FileWatcherProvider {
             watcherOptions.usePolling = false;
         }
 
-        const excludes: string[] = ['**/__pycache__/**'];
+        const excludes: string[] = ['**/node_modules/**', '**/__pycache__/**', '**/.*/**'];
```

Comment:
Doesn't this exclude everything?

'**/.*/**'

Reply:
oh no, it excludes any `.` folders, like `.git`. That seems okay?

Reply:
Yeah, I just copied from [here](https://github.com/microsoft/pyright/blob/0a67c65bef4ce95d4de8d2218a0bd875d2fb9230/packages/pyright-internal/src/analyzer/service.ts#L580)